frontend/src/ui/Blocks/index.tsx

```diff
@@ -64,6 +66,9 @@ export const Block: React.FC<BlockProps> = ({ block: blockRef, realm }) => {
 
     return <div>
         {match(__typename, {
+            "TitleBlock": () => <TitleBlock
+                fragRef={block}
+            />,
```

Comment:
I wouldn't mind having this (and `Text` and `Video` blocks below) in one line...